src/main/webapp/app/overview/course-card.component.ts

```diff
@@ -109,4 +109,13 @@ export class CourseCardComponent implements OnChanges {
         }
         this.nextExerciseDueDate = nextExerciseDueDate;
     }
+
+    /**
+     * Returns the background color of the course as a variable
+     */
+    get hoverEffectForBackground(): Record<string, string> {
+        return {
+            '--background-color-for-hover': this.course.color || this.ARTEMIS_DEFAULT_COLOR,
+        };
+    }
```

Comment:
Can you please set this as a variable in the `onChanges()` method. The getter violates our rule to not use functions in templates. => Constant rechecks.

Reply:
I agree. In addition, this is duplicated code with `course-management-card.component.ts` which we should try to avoid. Please try to find a more generic solution